doc/ug/getting_started_dv.md

```diff
@@ -61,28 +60,28 @@ There is support for Cadence Xcelium as well, which is being slowly ramped up.
 
 ## Building and Running Tests
 
-The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<ip>_sanity_vseq.sv` for developing the sanity test.
-The sanity test can be run as-is by invoking `make`, as a "hello world" step to bring the DUT out of reset.
+The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<dut>_sanity_vseq.sv` for developing the sanity test.
+The sanity test can be run as-is by invoking `dvsim.py`, as a "hello world" step to bring the DUT out of reset.
 ```console
 $ cd path-to-dv
-$ make [SIMULATOR=xcelium] [WAVES=1]
+$ util/dvsim.py <dut>_sim_cfg.hjson -i <dut>_sanity [-w] [-t xcelium]
 ```
 
 The generated initial testbench is not expected to compile and elaborate successfully right away.
 There may be additional fixes required, which can be hopefully be identified easily.
 Once the testbench compiles and elaborates without any errors or warnings, the sanity sequence can be developed further to access a major datapath and test the basic functionality of the DUT.
 
 VCS is used as the default simulator.
-It can be switched to Xcelium by setting `SIMULATOR=xcelium` on the command line.
-The `WAVES=1` switch will cause an `fsdb` dump to be created from the test.
+It can be switched to Xcelium by setting `-t xcelium` on the command line.
+The `-w` switch will cause an `fsdb` dump to be created from the test.
 The Synopsys Verdi tool can be invoked (separately) to debug the waves.
 Please refer to the [DV simulation flow]({{< relref "hw/dv/tools/README.md" >}}) for additional details.
 
 The `uvmdvgen` script also provides a CSR suite of tests which can be run right out of the box.
```

Comment:
```suggestion
The `uvmdvgen` script also enables the user to run the full suite of CSR tests, if the DUT does have CSRs in it. 
They can be run right out of the box:
```

Reply:
Thanks Sri for the detailed suggestion. Updated!